modules/mesh_com/package.sh

```diff
@@ -5,6 +5,8 @@ if [ "${ROS_DISTRO}" = "" ]; then
    exit 1
 fi
 
+export DEB_BUILD_OPTIONS=nocheck
```

Comment:
This could use a code comment explaining why

Reply:
Removed the nocheck build option. Build works without it. I must have added some specific changes when it failed, but now it seems to work fine anyway, so no extra option needed.